src/components/text-input/code-sequence/index.njk

```diff
@@ -15,5 +15,6 @@ layout: layout-example.njk
   classes: "govuk-input--width-5 govuk-input--extra-letter-spacing",
   id: "authentication-code",
   name: "authentication-code",
-  spellcheck: false
+  spellcheck: false,
+  value: "NC1701"
```

Comment:
🖖

```suggestion
  value: "NCC-1701"
```

Reply:
As much as I'd love to, I felt keeping within the six character restriction of actual company authentication codes was more important 😜